modules/microshift-nw-create-http-based-route.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-configuring-routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-creating-a-route_{context}"]
+= Creating an HTTP-based route
+
+A route allows you to host your application at a public URL. It can either be secure or unsecured, depending on the network security configuration of your application. An HTTP-based route is an unsecured route that uses the basic HTTP routing protocol and exposes a service on an unsecured application port.
+
+The following procedure describes how to create a simple HTTP-based route to a web application, using the `hello-microshift` application as an example.
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to your {microshift-short} cluster.
+* You have a web application that exposes a port and a TCP endpoint listening for traffic on the port.
+
+.Procedure
+
+. Create a service called `hello-microshift` by running the following command:
++
+[source,terminal]
+----
+$ oc expose pod hello-microshift -n $namespace
+----
+
+. Create an unsecured route to the `hello-microshift` application by running the following command:
++
+[source,terminal]
+----
+$ oc expose svc/hello-microshift --hostname=microshift.com $namespace
+----
+
+.Verification
+
+* To verify that the `route` resource that you created, run the following command:
++
+[source,terminal]
+----
+$ oc get routes -o yaml <name of resource> -n $namespace <1>
+----
+<1> In this example, the route is named `hello-microshift` and the namespace is named `hello-microshift`.
+
+.Sample YAML definition of the created unsecured route:
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: hello-microshift
+  namespace: hello-microshift
+spec:
+  host: microshift.com <1>
+  port:
+    targetPort: 8080 <2>
+  to:
+    kind: Service
+    name: hello-microshift
+----
+<1> Example hostname.
+<2> `targetPort` is the target port on pods that is selected by the service that this route points to.
```

Comment:
Is there a simpler way to state this? It seems a bit word-salad, if you don't mind my saying.

```suggestion
<2> `targetPort` is the port on pods that your service uses.
```

Reply:
Yeah, it is. But it's accurate. Lemme chew on it a bit more.